distributed/utils.py

```diff
@@ -386,7 +386,7 @@ def key_split_group(x):
     typ = type(x)
     if typ is tuple:
         return x[0]
-    elif typ is str:
+    elif typ is unicode or typ is str:
```

Comment:
Can we time this for the common cases of `('x', 0, 1)` and `'x-123'` and check that things don't slow down too much?  If not then lets just go with this.